src/strongbox/github_api.clj

```diff
@@ -120,51 +129,58 @@
+                  ;; game track present in release name, prefer that over `:known-game-tracks`
+                  release-game-track {:game-track release-game-track, :version (track-version version release-game-track) :-mo :release-in-asset-name}
 
-                  ;; single asset, no game track present in file name, multiple known game tracks. assume all game tracks supported
-                  (and single-asset? many-game-tracks?) [{:game-track :classic :version (track-version version :classic) :-mo :sa--Ngt}
-                                                         {:game-track :retail :version version :-mo :sa--Ngt}]
+                  ;; single asset, no game track present in asset name or release name and no `:game-track-list`. assume `:retail`
+                  (and single-asset? no-known-game-tracks?) (do (debug (format "no game track detected for release '%s' and asset '%s', assuming 'retail'"
+                                                                               (:name release) (:name asset)))
+                                                                {:game-track :retail :version version :-mo :sa--ngt})
 
-                  ;; multiple assets, no game track present in file name, no known game tracks. default to :retail
-                  ;; ambiguous case, other assets may have game track in their file name
-                  (and many-assets? no-known-game-tracks?) {:game-track :retail :version version :-mo :ma--ngt}
+                  ;; single asset, no game track present in asset name or release name and just a single entry in `:game-track-list`. use that.
+                  (and single-asset? single-game-track?) {:game-track (first known-game-tracks)
+                                                          :version (track-version version (first known-game-tracks))  :-mo :sa--1gt}
 
-                  ;; multiple assets, no game track present in file name, single known game track. use that.
-                  ;; ambiguous case, other assets may have game track in their file name
-                  ;; this or other assets may be variations of the 'main' addon, like '-nolib' ?
+                  ;; single asset, no game track present in asset name or release name with multiple entries in `:game-track-list`.
+                  ;; assume all entries in `:game-track-list` supported.
+                  (and single-asset? many-game-tracks?) (vec (for [game-track known-game-tracks]
+                                                               {:game-track game-track, :version (track-version version game-track) :-mo :sa--Ngt}))
+
+                  ;; multiple assets, no game track present in asset name or release name, and no `:game-track-list`.
+                  ;; default to `:retail` and make a note that we're guessing in the face of ambiguity.
+                  ;; other assets may have a game track in their asset name.
```

Comment:
make a note that this is identical logic for single-asset? + no-known-game-tracks